recipes/ligo-segments/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ligo-segments" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 464314cff3d37307b6044bddc02b051f
+  sha256: eab881e7646de8fb47ad7f602a8149488e938af73d5f639e65b7dfa2b7180657
+  patches:
+    - pure-python-windows.patch  # [win]
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
I think you need to add a `{{ compiler('c') }}` dependency: https://conda-forge.org/docs/meta.html?#compilers